### ISSUE TITLE
working on how to handle data importing

### ISSUE THREAD

**Issue by vanadgir** [vanadgir/dndler#2]
Option 1] figure out .csv import as array or object
Option 2] convert the existing .csv into .json

**Comment by grahambleon** [comment]
I vote option 2.  Seems like it'll be easier to deal with in the long run

**Comment by br-ndt** [comment]
Agreed. We may also want to consider some kind of true database like Mongo to handle both the input data and created characters. For now, I think .json is a better approach

**Comment by vanadgir** [comment]
Yep, going with option 2. Going to make a python script to read csv > turn into a dict > turn dict into json.

Definitely want to use an actual database down the line.

**Comment by grahambleon** [comment]
closing this because we settled on a plan
